### PR TITLE
fix install command

### PR DESCRIPTION
**Pull request by Wrymouth**

### CODE REVIEW

File: src/package/init.rs [hunk 1]
```diff
@@ -269,11 +269,19 @@ impl InitialisedPackage {
...
+                bail!("failed to run decomp install_deps.sh");
+            }
+
+            let status_install = Command::new("bash")
+                .arg("install_compilers.sh")
+                .current_dir(package.path().join(SUBREPO_DIR_NAME))
+                .status()?;
+            if !status_install.success() {
+                bail!("failed to run decomp install_compilers.sh");
             }
```
Comment (nanaian) [inline src/package/init.rs:285]: In Rust you can shadow variables like this:

```suggestion
            let status = Command::new("bash")
                .arg("install_compilers.sh")
                .current_dir(package.path().join(SUBREPO_DIR_NAME))
                .status()?;
            if !status.success() {
                bail!("failed to run decomp install_compilers.sh");
            }
```

**Review by nanaian — CHANGES_REQUESTED** [review]
Thanks!

Ideally we wouldn't break support for older versions of decomp before this change was made. Can you add a check to see if a `install.sh` file exists, and if so, run that instead of these two?

**Comment by nanaian** [comment]
This bug introduced by https://github.com/pmret/papermario/commit/8837fbdf65386832a453ff15f07a216690222095

**Comment by Wrymouth** [comment]
single_dependency appears to fail

**Comment by nanaian** [comment]
That's fine, it fails on `main` too 🤪

**Pull request merged**